extensions/arc/package.json

```diff
@@ -577,7 +577,10 @@
                           "type": "text",
                           "defaultValue": "sqlinstance1",
                           "required": true,
-                          "labelWidth": "100%"
+                          "labelWidth": "100%",
+                          "textValidationRequired": true,
+                          "textValidationRegex": "^[a-z0-9]{1}([-a-z0-9]{0,11}[a-z0-9])?$",
```

Comment:
I'm pretty sure it can't start with a number, but I'm verifying.  (I think the azdata validation is also wrong on this)

Reply:
Yeah, azdata validation is wrong.  Can't start with a number.